src/sdk/model/company/__json__/o365-org-create-request-json.ts

```diff
@@ -5,6 +5,9 @@ export interface O365OrgCreateRequestJson {
   name: string;
   account: string;
   password: string;
+  use_modern_auth: boolean;
+  application_id: string;
```

Comment:
since this is only conditionally required, should either use `?` to make it optional, or make `null` a possible type. perhaps `string | null` would be best since that's how `repository_id` is typed